src/modules/roc_core/target_posix/roc_core/fast_random.h

```diff
@@ -20,9 +20,9 @@ namespace core {
 //! Get random integer.
 //! Thread-safe.
 //! @returns random value in range [from; to].
-uint32_t random(uint32_t from, uint32_t to);
+uint32_t fast_random(uint32_t from, uint32_t to);
```

Comment:
It's worth to mention in the comment, that it uses non-choreographically-strong, but faster PRNG.